src/use-cases/hybrid-search.md

```diff
@@ -0,0 +1,122 @@
+# Hybrid Search
+
+We will show you how to use `pgvecto.rs` to implement hybrid search in this section.
+
+## Vector Search and Full Text Search
+
+Postgres provides [full text search](https://www.postgresql.org/docs/current/textsearch-intro.html), based on [the match operator `@@`](https://www.postgresql.org/docs/current/textsearch-intro.html#TEXTSEARCH-MATCHING). It returns true if a `tsvector` (document) matches a `tsquery` (query). It doesn't matter which data type is written first:
+
+```sql
+SELECT 'a fat cat sat on a mat and ate a fat rat'::tsvector @@ 'cat & rat'::tsquery;
+ ?column?
+----------
+ t
+
+SELECT 'fat & cow'::tsquery @@ 'a fat cat sat on a mat and ate a fat rat'::tsvector;
```

Comment:
Same. Add some comments in SQL to indicate the meaning of the query. And what does the result mean